### ISSUE TITLE
[Feature Request] Improve color options in Filament Setup

### ISSUE THREAD

**Issue by neogenix** [bambulab/BambuStudio#661]
Hi!

Today, we have a limited number of colors, it'd be nice to either give us a few options like a color wheel, or just increase the color options (and include metalic, etc...)

**Comment by aridShrub** [comment]
I agree ; metallic, translucent filaments need more than just a colour patch.

I would like to see the Bambu Lab colour names, e.g. “PLA Matte Ivory White” ; vs. “PLA Basic White” both display “PLA” and a white colour bar, so you cannot differentiate between the two. Maybe if you hover over it , additional detail could be given.

There is a Bambu Lab transparent” filament” - PC Black - and it is listed as “Black” which is confusing.

I’d like to be able to enter an RGB for third party filament. I use the Apple colour names now -  pretty good but not “peachy”

![3D9DCD9C-E07E-4978-AA6E-9914C7CADA9E](https://user-images.githubusercontent.com/50304316/202819192-8b03fb85-6d33-4d6d-8d42-97df25a4f732.jpeg)

**Comment by aridShrub** [comment]
Bambu Lab PC "Black" filament is actually transparent.
